web/modules/webspark/webspark_utility/src/Drush/Commands/WebsparkUtilityCommands.php

```diff
@@ -0,0 +1,160 @@
+      return;
+    }
+    $backupDate = $getBackupDate ? unserialize($getBackupDate, ['allowed_classes' => TRUE])['date'] : NULL;
+    if (empty($backupDate) || ($backupDate < ($now - 86400))) {
+      // Create new backup if it doesn't exist or latest is older than 24 hours.
+      $result = shell_exec('terminus backup:create webspark-release-stable.dev --element=db');
+    }
+    // Get latest backup.
+    shell_exec('terminus backup:get webspark-release-stable.dev --element=db --to=../webspark-db-backup.sql.gz');
+
+    // Create snapshot of current database.
+    exec('mysqldump -uroot -proot db | gzip > ../webspark-db-snapshot' . $now . '.sql.gz');
+    if (is_file('../webspark-db-snapshot' . $now . '.sql.gz')) {
+      $this->io()->writeln(PHP_EOL . ' // Creating snapshot of current database...' . PHP_EOL);
+    }
+
+    // Import latest webspark release stable database into DDEV.
+    $this->io()->writeln(' // Importing database from Webspark Release Stable...' . PHP_EOL);
+    exec('zcat ../webspark-db-backup.sql.gz | mysql -uroot -proot db');
+
+    // Import hook updates.
+    $this->io()->writeln(' // Running update.php...' . PHP_EOL);
+    exec('drush updb -y');
+
+    $this->io()->writeln(PHP_EOL . ' // Updating configs...' . PHP_EOL);
+    // Get array of modules that have a config/install subdirectory.
+    $websparkPaths = (array) (glob('modules/webspark/*/config/install', GLOB_ONLYDIR)) ? glob('modules/webspark/*/config/install', GLOB_ONLYDIR) : [];
+    // Add webspark profile configs to the array.
+    $websparkPaths[] = 'profiles/webspark/webspark/config/install';
+    // Export all configs per path, remove untracked files, & update remaining.
+    foreach ($websparkPaths as $path) {
+      $this->logger()->info('Updating YAML files at ' . $path);
+      $self = Drush::aliasManager()->getSelf();
+      // Run drush config:export on @self with destination set to the path.
+      $process = $this->processManager()->drush($self, 'config:export', [], ['destination' => $path]);
+      $process->run();
+      // Remove untracked files from directory.
+      $output = shell_exec('git clean -df -- ' . $path);
+      // Handle output.
+      switch (TRUE) {
+        // Successful operation yields output beginning with "Removing".
+        case preg_match('/^Removing.*$/m', $output):
+          // Set directory glob path for recursive regex operations.
+          $dir = $path . '/*';
+          // Loop through files in directory.
+          foreach (glob($dir) as $filename) {
+            $file = file_get_contents($filename);
+            // Delete UUID and default config hash.
+            $file = preg_replace('/^uuid:\s[0-9a-z\-]+\n/m', '', $file);
+            $file = preg_replace('/^_core:\n\s+default_config_hash:\s\'?[a-zA-Z0-9\_\-]+\'?\n?/m', '', $file);
+            file_put_contents($filename, $file);
+            $filesToSkip = [
+              'profiles/webspark/webspark/config/install/block.block.asubrandheader.yml',
```

Comment:
I'm thinking we need to skip the input format fields here, too, since they are used on install, and then with special care in updates when needed, but generally we don't want to rewrite those.